kura/examples/org.eclipse.kura.example.gpio.led/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Configurable Led Example
+Bundle-SymbolicName: org.eclipse.kura.example.gpio.led
+Bundle-Version: 1.0.0.qualifier
+Automatic-Module-Name: org.eclipse.kura.example.gpio.led
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Import-Package: org.eclipse.kura.configuration;version="1.2.0",
```

Comment:
This should be a strict interval [1.2,1.3) as you are implementing a configurableComponent